setup.cfg

```diff
@@ -25,6 +25,12 @@ classifier =
     Programming Language :: Python :: 3.8
     Topic :: Security
 
+[extras]
```

Comment:
Is this the correct syntax?

https://setuptools.readthedocs.io/en/latest/setuptools.html#declaring-extras-optional-features-with-their-own-dependencies

Reply:
Good catch! Thank you. I've fixed it as it is shown in https://github.com/pypa/sampleproject/pull/57/files and https://setuptools.readthedocs.io/en/latest/setuptools.html#configuring-setup-using-setup-cfg-files